frontend/src/pages/Page/Page.tsx

```diff
@@ -41,14 +41,23 @@ export function Page({ children }: Props & React.HTMLProps<HTMLDivElement>) {
     remoteCss = classnames(css.full, css.default)
   }
 
+  // only show one message at a time
+  let snackbar = errorMessage
```

Comment:
could you create a switch? 
it's hard to read 4 conditions

Reply:
Can't do switch, but can do series of ifs